addons/main/stringtable.xml

```diff
@@ -587,6 +587,9 @@
             <English>Driver only</English>
             <German>Nur Fahrer</German>
         </Key>
+        <Key ID="STR_dui_show_dir_opt5">
+            <English>If the player has GPS</English>
```

Comment:
Looked through the PR now fully, looking good, just think the wording here needs to be align with the other entries a little more.

```suggestion
            <English>Only with GPS</English>
```

Reply:
Yeah, sounds good. I'll change it.